src/tree_schema.c

```diff
@@ -4454,11 +4454,10 @@ lys_make_implemented_r(struct lys_module *module, struct unres_schema *unres)
 
     for (i = 0; i < module->augment_size; i++) {
         /* apply augment */
-        if (apply_aug(&module->augment[i], unres)) {
+        if ((module->augment[i].flags & LYS_NOTAPPLIED && module->augment[i].target) && apply_aug(&module->augment[i], unres)) {
             return EXIT_FAILURE;
         }
     }
-
```

Comment:
No reason to remove the empty line so please keep it there (I would let this one slide but since you will be correcting other things... :) ).